tests/cases/fourslash/codeFixUnusedIdentifier_all_delete_js.ts

```diff
@@ -0,0 +1,38 @@
+/// <reference path='fourslash.ts' />
+
+// @allowJs: true
+
+// @Filename: /a.js
+/////** @template T */
+////function f() {}
+////
+/////**
+//// * Doc
+//// * @template T
```

Comment:
Can other things appear on this line (i.e. after `T`)?  Skimming the code above, it looks like that's handled, but I didn't notice a test for it.